resources/views/Widgets/Common/LinkWidget.json

```diff
@@ -11,29 +11,29 @@
         "appearance": {
             "type": "select",
             "required": true,
-            "defaultValue": "none",
+            "defaultValue": "primary",
             "options": "@include(optionalAppearance)"
         },
         "size": {
             "type": "select",
             "required": false,
-            "defaultValue": "md",
+            "defaultValue": "",
             "options":{
                 "name": "Widget.linkSizeLabel",
                 "tooltipText": "Widget.linkSizeTooltip",
                 "listBoxValues": [
                     {
-                        "value": "sm",
+                        "value": "btn-sm",
```

Comment:
In anderen Widget verwenden wir die Werte "sm", "md" und "lg". Sollten wir das nicht einheitlich machen, für den fall, dass wir es irgendwann zusammenführen wollen?

Reply:
Im Widget wird das kombiniert als btn-{{ size }}. Damit lässt sich die Standardgröße (ohne Klasse) nicht abbilden. 

Entweder machen wir alle einheitlich auf 
btn-sm / ohne Klasse / btn-lg
-> Alle Buttons außerhalb von Widgets sind derzeit ohne Klasse.

Oder alle auf:
sm / md / lg und setzen es im Twig über btn-{{ size }}
-> btn-md ist allerdings größer als ohne Klasse.

Reply:
> Oder alle auf:
> sm / md / lg und setzen es im Twig über btn-{{ size }}
> -> btn-md ist allerdings größer als ohne Klasse.

Das finde ich eine gute Idee.

Reply:
Dies ist die korrekte Verwendung. Wird bei anderen Buttons angeglichen. Karte nehme ich auf.